doc/Writerside/topics/starter-topic.md

```
@@ -0,0 +1,12 @@
+# Подсистема интеграции.
+
+Данная подсистема интеграции позволяет быстро и просто организовать событийную интеграцию между 1с и другими информационными системами.
+
+## Основные преимущества.
+
+* **Простота внедрения.** Для внедрения и организации первого потока данных - требуется буквально 3 простых действия.
+* **Скорость работы.** Данная подсистема способна обрабатывать больше тысячи сообщений в минуту.
+* **Соблюдение контрактов.** В подсистеме включен блок валидации сообщений как входящих, так и исходящих. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)